src/components/layout/Tutorial/Step/CreateOffer.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CreateOffer = () => (
+  <div>
+    <h1>
```

Comment:
```suggestion
    <h1 id="tuto-dialog-label">
```
cf. commentaire à propos de labelledBy dans TutorialModal.jsx